articles/sql-database/sql-database-connectivity-architecture.md

```diff
@@ -45,14 +45,17 @@ If you are connecting from outside Azure, your connections have a connection pol
 ![architecture overview](./media/sql-database-connectivity-architecture/connectivity-from-outside-azure.png)
 
 > [!IMPORTANT]
-> When using service endpoints with Azure SQL Database your policy is **Redirect** by default. So to enable connectivity from inside your Vnet you must allow outbound to all Azure SQL Database IP addresses, not just the gateway IPs. This can be done with the help of NSG (Network Security Group) Service Tags, if you want to allow outbound only to gateway IPs please change your setting to **Proxy**.
+> When using service endpoints with Azure SQL Database your policy is **Proxy** by default. So to enable connectivity from inside your Vnet you must allow outbound to Azure SQL Database Gateway IP addresses specified in the list below.
```

Comment:
In second sentence, please change to "To enable connectivity from inside your Vnet, allow outbound connections to the Azure SQL Database Gateway IP addresses specified in the following list: